app/frontend/styles/_primary-navigation.scss

```diff
@@ -2,30 +2,17 @@
   background-color: govuk-colour("light-grey");
 }
 
-.app-primary-navigation__container {
-  @include govuk-width-container;
-  font-size: 0; // Hide whitespace between elements
-  text-align: justify; // Trick to remove the need for floats
-
-  &:after {
-    content: '';
-    display: inline-block;
-    width: 100%;
-  }
-
-}
-
 .app-primary-navigation__list {
+  @include govuk-clearfix;
   list-style: none;
   margin: 0;
   padding: 0;
-  @include govuk-clearfix;
 }
 
 .app-primary-navigation__item {
   @include govuk-font($size: 19);
-  margin-right: govuk-spacing(3);
   float: left;
```

Comment:
Comment above says that `justify` is used to remove the need for floats.

Reply:
Having removed that, this is needed to allow for right floated navigation items.